tests/cbmc/proofs/s2n_hash_digest_size/s2n_hash_digest_size_harness.c

```diff
@@ -27,18 +27,6 @@ void s2n_hash_digest_size_harness()
 
     /* Operation under verification. */
     if (s2n_hash_digest_size(alg, out) == S2N_SUCCESS) {
-        /* Post-conditions. */
-        switch (alg) {
-        case S2N_HASH_NONE:     assert(*out == 0);                    break;
-        case S2N_HASH_MD5:      assert(*out == MD5_DIGEST_LENGTH);    break;
-        case S2N_HASH_SHA1:     assert(*out == SHA_DIGEST_LENGTH);    break;
-        case S2N_HASH_SHA224:   assert(*out == SHA224_DIGEST_LENGTH); break;
-        case S2N_HASH_SHA256:   assert(*out == SHA256_DIGEST_LENGTH); break;
-        case S2N_HASH_SHA384:   assert(*out == SHA384_DIGEST_LENGTH); break;
-        case S2N_HASH_SHA512:   assert(*out == SHA512_DIGEST_LENGTH); break;
-        case S2N_HASH_MD5_SHA1: assert(*out == MD5_DIGEST_LENGTH + SHA_DIGEST_LENGTH); break;
-        default:
-            __CPROVER_assert(0, "Unsupported algorithm.");
-        }
+        assert(*out <= S2N_MAX_DIGEST_LEN);
```

Comment:
I could have just updated this switch statement instead, but I really don't think duplicating the entire contents of the method under test is useful. Really the only safety assertion we need to make is that the digest size isn't unexpectedly large.